values.yaml

```diff
@@ -2,8 +2,8 @@
 # task steps container images
 images:
   bash: registry.access.redhat.com/ubi8-minimal:8.9
-  skopeo: registry.access.redhat.com/ubi8/skopeo:8.9
-  buildah: registry.access.redhat.com/ubi8/buildah:8.9
+  skopeo: registry.access.redhat.com/ubi8/skopeo:8.10-5
+  buildah: registry.access.redhat.com/ubi8/buildah:8.10-5
```

Comment:
@vdemeester we got a notification for CVE https://redhat-internal.slack.com/archives/C050H792L2Y/p1710788197878449 but I don't see that particular CVE fix in any of rhel9 or rhle8 so i bumped that latest tag is that fine ?

Reply:
Yes I think that's fine 👍🏼